src/common/schema/application.js

```diff
@@ -395,6 +395,9 @@ module.exports = {
     veteranOtherIncome: ['understandsFinancialDisclosure'],
     spouseGrossIncome: ['understandsFinancialDisclosure'],
     spouseNetIncome: ['understandsFinancialDisclosure'],
-    spouseOtherIncome: ['understandsFinancialDisclosure']
+    spouseOtherIncome: ['understandsFinancialDisclosure'],
```

Comment:
That was the intent-- `spouseOtherIncome` relies on `understandsFinancialDisclosure`.  However, you're totally right that this doesn't do anything (other than see if understandsFinancialDisclosure exists, which it always does).

The right way to do it would be to actually see if `understandsFinancialDisclosure` is `true` or not, like I'm doing in [this PR](https://github.com/department-of-veterans-affairs/healthcare-application/pull/274).